pkg/autoupdate/tuf/ci/tuf_server.go

```diff
@@ -17,6 +15,34 @@ import (
 	"github.com/theupdateframework/go-tuf"
 )
 
+var (
+	//go:embed testdata/darwin_launcher.tar.gz
+	darwinLauncher []byte
```

Comment:
Feels like this might have been cleaner with the embed.FS

Reply:
You mean embedding the entire testdata dir with embed.FS, or something else? I can take a look!

Reply:
Updated and I like it better! That also sped up the tests even more